zipkin-ui/js/skew.js

```diff
@@ -68,6 +68,11 @@ class Node {
   }
 }
 
+// In javascript, dict keys can't be objects
+function keyString(id, shared) {
```

Comment:
notable (id, shared) is not enough. This has to be kindof different. will think on a plane trip tonight about it